include/zephyr/drivers/i2c.h

```diff
@@ -531,6 +531,15 @@ static inline void i2c_dump_msgs(const struct device *dev, const struct i2c_msg
 	i2c_dump_msgs_rw(dev, msgs, num_msgs, addr, false);
 }
 
+/** @cond INTERNAL_HIDDEN */
+
+static inline void i2c_set_last_msg_stop(struct i2c_msg *msgs, uint8_t num_msgs)
+{
+	msgs[num_msgs - 1].flags |= I2C_MSG_STOP;
+}
+
+/** @endcond */
```

Comment:
is this inline helper really necessary?

Reply:
no, but it is pretty :)

Reply:
`msgs[num_msgs - 1].flags |= I2C_MSG_STOP;` is straight forward.
I don't see the point of adding 8 lines of code if there is only one call to the inline function.

Reply:
Understood :)